svo/include/svo/global.h

```diff
@@ -25,6 +25,10 @@
 #include <math.h>
 
 #include <Eigen/Core>
+template <class T, class Allocator =  Eigen::aligned_allocator<std::pair<const int,T>>   >
```

Comment:
Rather than create a new vector type, using the Eigen macro would be sufficient:
```
#include<Eigen/StdVector>
EIGEN_DEFINE_STL_VECTOR_SPECIALIZATION(Vector3d)
```

This change is not intrusive (I mean you don't new to change vector to vectorA)

For the seed list, since EIGEN_MAKE_ALIGNED_OPERATOR_NEW is already used in the seed class you should not to have to do a fix.